### PR TITLE
REAL-25

### PR DESCRIPTION
**Pull request by thienngole**
Dummy endpoints for users.

https://gouda.msudenver.edu/jira/browse/REAL-25

### CODE REVIEW

File: onthequad/onthequad/User.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class User:
+    def __init__(self, id, firstname, lastname, email, username, photoURL, events ):
+        self.id = id
+        self.firstname = firstname
+        self.lastname = lastname
+        self.email = email
+        self.username = username
+        self.photoURL = photoURL
+        self.evetns = events
```
Comment (paugajt) [inline onthequad/onthequad/User.py]: events is misspelled

File: onthequad/onthequad/users.py [hunk 2]
```diff
@@ -9,3 +9,5 @@ def on_get(self, req, resp):
     def on_post(self, req, resp):
         print("Removing all them red lines")
 
+
```
Comment (paugajt) [inline onthequad/onthequad/users.py]: extra spaces

File: onthequad/onthequad/dummyusers.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                'msg': "there is nothing to update"
+            }
+            resp.body = json.dumps(sendback)
+            resp.status = falcon.HTTP_400
+        else:
+            sendback = {
+                'msg': "Stuff will be updated"
+            }
+            resp.body = json.dumps(sendback)
+            resp.status = falcon.HTTP_200
```
Comment (JohnSanders92) [inline onthequad/onthequad/dummyusers.py]: Coding conventions say to leave an empty line at the end of the file.

File: onthequad/onthequad/app.py [hunk 4]
```diff
@@ -19,6 +20,8 @@
 api = application = falcon.API
```
Comment (JohnSanders92) [inline onthequad/onthequad/app.py]: This needs to be updated from master

**Pull request merged**